examples/src/main/python/sort.py

```diff
@@ -23,9 +23,11 @@
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 2:
-        print("Usage: sort <file>", file=sys.stderr)
-        exit(-1)
+    """
+    Usage: sort <file>
+    """
+
+    filename = "../resources/people.txt" if len(sys.argv) != 2 else sys.args[1]
```

Comment:
you probably would have one check `if len(sys.argv) > 1` and then set all parameter in the if block - we can assume if any parameter is there the user needs to specify all of it?